com.unity.shadergraph/Documentation~/Sample-Reflected-Cubemap-Node.md

```diff
@@ -0,0 +1,24 @@
+# Sample Cubemap Node
+
+## Description
+
+Samples a **Cubemap** with reflected vector and returns a **Vector 4** color value for use in the shader. Requires **View Direction** and **Normal** inputs to sample the **Cubemap**. You can achieve a blurring effect by sampling at a different Level of Detail using the **LOD** input. You can also define a custom **Sampler State** using the **Sampler** input.
+
```

Comment:
probably worth noting that it was previously Sample Cubemap?

Reply:
I will let the reply to @sharlenet . I don't know. I think this is the purpose of the upgrade guide to say that?
I think the doc should be what is the current state of node. But open to change if needed.